frontend/river-names-decoder/style.css

```
@@ -0,0 +1,108 @@
+*{margin:0;padding:0;box-sizing:border-box;font-family:'Outfit',-apple-system,BlinkMacSystemFont,'Segoe UI',sans-serif}
```

Stylelint (17.14.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)